### PR TITLE
Docs 4725 april30 rtf support

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/monitor-applications-on-rtf.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Monitoring for applications deployed to Runtime Fabric is enabled by default.
+
+== Supported Mule Versions
+
+Anypoint Monitoring supports the following Mule versions for Runtime Fabric deployments:
+
+* Mule Runtime 3.8.7
+* Mule Runtime 3.9.1
+* Mule Runtime 3.9.2
+* Mule Runtime, version 4.1.1+
```
Comment (sundippatel) [inline modules/ROOT/pages/monitor-applications-on-rtf.adoc]: Please update to 4.1.2+ (RTF doesn't support Mule 4.1.1)

File: modules/ROOT/pages/performance-and-impact.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+= Anypoint Monitoring Performance and Impact
+
+These are the findings from Anypoint Monitoring performance testing.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/performance-and-impact.adoc]: I don't know what this means?

File: modules/ROOT/pages/performance-and-impact.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Expected CPU utilization impact: 3-5%
+
+== Data Retention and Aggregation
+
+* Minute Resolution for Metrics
+* Free tier (non-Titanium)
+ ** Metrics retention is 30 days for multi-tenant
+
+== Screen Resolution
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/performance-and-impact.adoc]: Feels like this should be a note or subhead of the intro, not after the rich content about what the product does. I know this is early draft, so not sure this is relevant.

File: modules/ROOT/pages/performance-and-impact.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
+= Anypoint Monitoring Performance and Impact
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/performance-and-impact.adoc]: Should the title be Anypoint Monitoring Performance Testing? Then the intro sentence  on line 3 could summarize "Review details about the performance impact and data retention with Anypoint Monitoring." or similar

**Pull request merged**